prow/oss/config.yaml

```diff
@@ -150,6 +150,13 @@ deck:
       github_orgs:
       - GoogleCloudPlatform
 
+in_repo_config:
+  enabled:
+    chaotoppicks: true
+
+  allowed_clusters:
+    "chaotoppicks": ["build"]
```

Comment:
If we're actually going to create a build cluster to test this, please name it something more specific than `build`.

Reply:
done